added mgr-storage-sever sections to vm and server deployment (#3161)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc

. Reboot the system.

=== Configure Custom Persistent Storage

This step is optional.
However, if custom persistent storage is required for your infrastructure, use the [command]``mgr-storage-proxy`` tool. 

* For more information, see [command]``mgr-storage-proxy --help``.

> This tool simplifies creating the container storage and database volumes.

Reviewer: ```suggestion
This tool simplifies creating the container storage and Squid cache volumes.
```

Use the command in the following manner: 
+

----

> mgr-storage-proxy <storage-disk-device> [<database-disk-device>]

Reviewer: guessing (and space for retail images?):

```suggestion
mgr-storage-proxy <storage-disk-device>
```

----

For example: 

----

> mgr-storage-proxy /dev/nvme1n1 /dev/nvme2n1

Reviewer: ```suggestion
mgr-storage-proxy /dev/nvme1n1
```
